File: lecture 1/sigma java/bubble_short.java
import javax.swing.*;

public class bubble_short {
    public static void bubblr(int arr[]){
        for (int i = 0; i < arr.length-1; i++) {
            for (int j = 0; j < arr.length - 1-i; j++) {
                if (arr[j] > arr[j + 1]) {
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
    }
    public static void main(String[] args) {
       int arr[] = {12,3,7,76,887,23};
       bubblr(arr);
        for (int i = 0; i < arr.length; i++) {
            System.out.println(arr[i]);
        }
    }
}

File: dsa pythan/pythonProject1/bubble_short.py
# def bubble_short(n):
#     for r in range(1,len(n)):
#         for i in range(len(n)-r):
#             if n[i] > n[i+1]:
#                 temp = n[i]
#                 n[i] = n[i+1]
#                 n[i+1] = temp

# def bubble_short(n):
#     for r in range(1,len(n)):
#         for i in range(len(n)-r):
#             if n[i] > n[i+1]:
#                 n[i],n[i+1] = n[i+1],n[i]
def bubble_short(n):
    flag = False
    for r in range(1,len(n)):
        flag = False
        for i in range(len(n)-r):
            if n[i] > n[i+1]:
                n[i],n[i+1] = n[i+1],n[i]
                flag = False
            if not flag:
                break

l = [67,23,78,99,11,34,77,93,98,33]
bubble_short(l)
print(l)